<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1076</width>
    <height>703</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Qt5 - Explorando QCamera</string>
  </property>
  <property name="accessibleName">
   <string/>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(211, 215, 207);</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="1" rowspan="3">
    <widget class="QGroupBox" name="webcam">
     <property name="maximumSize">
      <size>
       <width>320</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="title">
      <string>                                  CONTROLES</string>
     </property>
     <widget class="QLineEdit" name="console">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>650</y>
        <width>211</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(238, 238, 236);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QSpinBox" name="servo_base">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>380</y>
        <width>91</width>
        <height>30</height>
       </rect>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:#fff</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
      <property name="suffix">
       <string> °</string>
      </property>
      <property name="minimum">
       <number>0</number>
      </property>
      <property name="maximum">
       <number>180</number>
      </property>
      <property name="value">
       <number>90</number>
      </property>
     </widget>
     <widget class="Line" name="line_4">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>310</y>
        <width>281</width>
        <height>20</height>
       </rect>
      </property>
      <property name="lineWidth">
       <number>2</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QPushButton" name="down_webcam">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>540</y>
        <width>41</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="imagens.qrc">
        <normaloff>:/down.png</normaloff>:/down.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>25</width>
        <height>25</height>
       </size>
      </property>
      <property name="autoRepeat">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLineEdit" name="distancia">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>610</y>
        <width>211</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(238, 238, 236);</string>
      </property>
      <property name="inputMask">
       <string/>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="left_webcam">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>480</y>
        <width>41</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="imagens.qrc">
        <normaloff>:/left.png</normaloff>:/left.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>25</width>
        <height>25</height>
       </size>
      </property>
      <property name="autoRepeat">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>360</y>
        <width>91</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-color: rgb(0, 0, 0);</string>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:8pt;&quot;&gt;Servo motor Cam&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="Line" name="line_3">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>590</y>
        <width>291</width>
        <height>20</height>
       </rect>
      </property>
      <property name="lineWidth">
       <number>2</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="Line" name="line">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>20</width>
        <height>541</height>
       </rect>
      </property>
      <property name="lineWidth">
       <number>2</number>
      </property>
      <property name="midLineWidth">
       <number>1</number>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>360</y>
        <width>111</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-color: rgb(0, 0, 0);</string>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:8pt;&quot;&gt;Servo motor Base&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QPushButton" name="up_webcam">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>410</y>
        <width>41</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="imagens.qrc">
        <normaloff>:/up.png</normaloff>:/up.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>25</width>
        <height>25</height>
       </size>
      </property>
      <property name="autoRepeat">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QSpinBox" name="servo_camera">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>380</y>
        <width>91</width>
        <height>30</height>
       </rect>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:#fff</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
      <property name="suffix">
       <string> °</string>
      </property>
      <property name="minimum">
       <number>0</number>
      </property>
      <property name="maximum">
       <number>180</number>
      </property>
      <property name="value">
       <number>84</number>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>330</y>
        <width>141</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Controle da  Camera&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QPushButton" name="right_webcam">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>480</y>
        <width>41</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="imagens.qrc">
        <normaloff>:/right.png</normaloff>:/right.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>25</width>
        <height>25</height>
       </size>
      </property>
      <property name="autoRepeat">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="carro">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>170</y>
        <width>81</width>
        <height>71</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image:url(:carro.jpg);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="down_carro">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>250</y>
        <width>41</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="imagens.qrc">
        <normaloff>:/down.png</normaloff>:/down.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>25</width>
        <height>25</height>
       </size>
      </property>
      <property name="autoRepeat">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="left_carro">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>180</y>
        <width>41</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="imagens.qrc">
        <normaloff>:/left.png</normaloff>:/left.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>25</width>
        <height>25</height>
       </size>
      </property>
      <property name="autoRepeat">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="up_carro">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>120</y>
        <width>41</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="imagens.qrc">
        <normaloff>:/up.png</normaloff>:/up.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>25</width>
        <height>25</height>
       </size>
      </property>
      <property name="autoRepeat">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="right_carro">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>180</y>
        <width>41</width>
        <height>41</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>41</width>
        <height>0</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="imagens.qrc">
        <normaloff>:/right.png</normaloff>:/right.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>25</width>
        <height>25</height>
       </size>
      </property>
      <property name="autoRepeat">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>60</y>
        <width>131</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Controle do Carro &lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="Line" name="line_5">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>40</y>
        <width>291</width>
        <height>20</height>
       </rect>
      </property>
      <property name="lineWidth">
       <number>2</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QSpinBox" name="pwm_ctrl">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>90</y>
        <width>91</width>
        <height>30</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:#fff</string>
      </property>
      <property name="suffix">
       <string> pwm</string>
      </property>
      <property name="minimum">
       <number>250</number>
      </property>
      <property name="maximum">
       <number>2500</number>
      </property>
     </widget>
     <widget class="QSpinBox" name="ms_ctrl">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>90</y>
        <width>83</width>
        <height>30</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:#fff</string>
      </property>
      <property name="suffix">
       <string> ms</string>
      </property>
      <property name="minimum">
       <number>2000</number>
      </property>
      <property name="maximum">
       <number>2500</number>
      </property>
     </widget>
     <widget class="Line" name="line_2">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>50</y>
        <width>20</width>
        <height>541</height>
       </rect>
      </property>
      <property name="lineWidth">
       <number>2</number>
      </property>
      <property name="midLineWidth">
       <number>1</number>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>460</y>
        <width>81</width>
        <height>71</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="imagens.qrc">
        <normaloff>:/webcam.jpg</normaloff>:/webcam.jpg</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>90</width>
        <height>80</height>
       </size>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>616</y>
        <width>67</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Distância</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>650</y>
        <width>67</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Usuário</string>
      </property>
     </widget>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>100</height>
      </size>
     </property>
     <property name="title">
      <string>botões</string>
     </property>
     <widget class="QPushButton" name="btnCaptureImage">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>30</y>
        <width>108</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Captura Image</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnClose">
      <property name="geometry">
       <rect>
        <x>630</x>
        <y>30</y>
        <width>80</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Sair</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnCloseCam">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>30</y>
        <width>93</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Desconectar</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnOpenCam">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>165</x>
        <y>30</y>
        <width>80</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Conectar</string>
      </property>
     </widget>
     <widget class="QComboBox" name="cbListCam">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>30</y>
        <width>111</width>
        <height>25</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="CaptureVideo">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>520</x>
        <y>30</y>
        <width>89</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>gravar vídeo</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>60</y>
        <width>571</width>
        <height>25</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(238, 238, 236);</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>60</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Rede Insp</string>
      </property>
     </widget>
    </widget>
   </item>
   <item row="0" column="0" rowspan="2">
    <widget class="QWidget" name="camViewfinder" native="true">
     <property name="styleSheet">
      <string notr="true">background-color:#555;</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="imagens.qrc"/>
 </resources>
 <connections/>
</ui>
